libsoftwarecontainer/component-test/filegateway_componenttest.cpp

```diff
@@ -69,3 +69,43 @@ TEST_F(FileGatewayTest, TestActivateWithMinimalValidConf) {
     ASSERT_TRUE(gw->setConfig(jsonConfig));
     ASSERT_TRUE(gw->activate());
 }
+
+TEST_F(FileGatewayTest, MalformedPath) {
+// Mount a path which is malformed, e.g. too many "/"
+    const std::string config =
+        "[{"
+        "  \"path-host\": \"//tmp//filename.txt\""
+        ", \"path-container\": \"///filename.txt\""
+        "}]";
+    loadConfig(config);
+
+    // Is this ok? Will the extra "/" will be removed?
+    ASSERT_TRUE(gw->setConfig(jsonConfig));
+    ASSERT_TRUE(gw->activate());
+}
+
+TEST_F(FileGatewayTest, MountNonExisting) {
+// Mount a non-existing path
```

Comment:
Is this a comment for what the test does? Put it above the test in that case, and comment on what's being tested, e.g. "Mount a non-existing path and assert that ... "